WARNING  urllib3.connectionpool:connectionpool.py:870 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001C49EE8A2E0>, 'Connection to 127.0.0.1 timed out. (connect timeout=1.0)')': /status
WARNING  urllib3.connectionpool:connectionpool.py:870 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001C49EE8AA60>, 'Connection to 127.0.0.1 timed out. (connect timeout=1.0)')': /status
WARNING  urllib3.connectionpool:connectionpool.py:870 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001C49EE95AF0>, 'Connection to 127.0.0.1 timed out. (connect timeout=1.0)')': /status
INFO     conftest:conftest.py:62 App opened without logging in.
INFO     utils.config:test_create_account.py:44 Clicked on 'Allow' button for notification permission.
INFO     utils.config:test_create_account.py:49 کلیک روی دکمه 'ایجاد حساب کاربری'...
INFO     utils.config:test_create_account.py:56 No login page displayed, continuing.
INFO     utils.config:test_create_account.py:60 کلیک روی دکمه 'بازکردن حساب'...
INFO     utils.config:test_create_account.py:63 دکمه 'بازکردن حساب' کلیک شد.
INFO     utils.config:test_create_account.py:67 انتخاب سرور 'UAT'...
INFO     utils.config:test_create_account.py:70 سرور 'UAT' انتخاب شد.
INFO     utils.config:test_create_account.py:74 کلیک روی دکمه 'شروع'...
INFO     utils.config:test_create_account.py:77 دکمه 'شروع' کلیک شد.
INFO     utils.config:test_create_account.py:82 Initial switch status: OFF
INFO     utils.config:test_create_account.py:88 کلیک بر روی تاگل سوییچ موافقت با قوانین و مقررات ...
INFO     utils.config:test_create_account.py:91 Switch status after toggle: ON
INFO     utils.config:test_create_account.py:97 کلیک روی دکمه تایید...
INFO     utils.config:test_create_account.py:99 دکمه تایید کلیک شد و به صفحه وارد کردن شماره تلفن هدایت شد.
INFO     utils.config:test_create_account.py:102 ورود شماره تلفن ...
INFO     utils.config:test_create_account.py:108 شماره تلفن وارد شد: 09134743454
INFO     utils.config:test_create_account.py:112 کلیک روی دکمه 'بعدی'...
INFO     utils.config:test_create_account.py:114 دکمه 'بعدی' کلیک شد.
INFO     utils.config:test_create_account.py:117 بررسی متن ریفرال...
INFO     utils.config:test_create_account.py:126 متن ریفرال صحیح است: مانند:‌ MDMGKP
INFO     utils.config:test_create_account.py:129 کلیک روی دکمه 'بعدی'...
INFO     utils.config:test_create_account.py:132 دکمه 'بعدی' کلیک شد.
INFO     utils.config:test_create_account.py:136 بررسی متن کد ملی...
INFO     utils.config:test_create_account.py:145 متن کد ملی صحیح است: کد ملی ۱۰ رقمی
INFO     utils.config:test_create_account.py:148 ورود کد ملی ...
INFO     utils.config:test_create_account.py:157 کدملی وارد شد: 6232968115
INFO     utils.config:test_create_account.py:166 کلیک روی دکمه 'بعدی'...
INFO     utils.config:test_create_account.py:168 دکمه 'بعدی' کلیک شد.
INFO     utils.config:test_create_account.py:173 سال با موفقیت اسکرول شد.
INFO     utils.config:test_create_account.py:176 کلیک روی دکمه 'بعدی'...
INFO     utils.config:test_create_account.py:178 دکمه 'بعدی' کلیک شد.